src/app/projects/components/project-settings/project-settings.component.html

```diff
@@ -81,6 +85,20 @@
                                             page="attributes" initialCaption="Registered">
                                         </kern-heuristic-statuses>
                                     </td>
+                                    <td class="whitespace-nowrap text-center px-3 py-2 text-sm text-gray-500">
+                                        <div class="inline-block">
+                                            <kern-dropdown [dropdownOptions]="{
+                                                optionArray: attributeVisibilityStates,
+                                                buttonCaption: attributeContainer.get('visibility').value,
+                                                valuePropertyPath: 'value',
+                                                hasFullWidth: true,
+                                                width: '170px'
+                                            }"
+                                                (optionClicked)="updateVisibilityAttributes($event, attributeContainer)">
```

Comment:
see above:
I'm assuming `attributeContainer.get('visibility').setValue($event)`

Reply:
I tried this and it works as expected, but when I change the value of the dropdown the expected value is not shown directly (ex. first HIDE then Hide like flickering)

Reply:
setting the explicit variable would help

Reply:
I fixed this by having two values: visibility and visibilityShow, one for the display in the dropdown and one for saving the value, is that okay?